API/Startup.cs

```diff
@@ -273,6 +274,17 @@ public void ConfigureServices(IServiceCollection services)
 
         app.UseForwardedHeaders();
 
+        var basePath = !Configuration.BaseUrl.StartsWith("/")
```

Comment:
Put all this logic in the setter, so that it always is set this way. you can also put in the getter to ensure it comes out this way as well in Configuration.